### PR TITLE
feat: Add avro doc field

### PR DESCRIPTION
**Pull request by mredjem**
Add support for Avro optional doc field based on SQL comments

### CODE REVIEW

File: src/main/java/com/at/avro/AvroField.java [hunk 1]
```diff
@@ -47,6 +52,14 @@ public Object getDefaultValue() {
     public boolean isDefaultValueSet() {
         return defaultValue != NOT_SET;
     }
+    
+    public String getDoc() {
+        return doc;
+    }
+    
+    public boolean isDocSet() {
+        return doc != null;
```
Comment (artur-tamazian) [inline src/main/java/com/at/avro/AvroField.java:61]: I think this is always true (based on test cases). Does column.getRemarks() return an empty string when there's no comment? I think we can change it to StringUtils.isNotBlank(doc). WDYT?

**Comment by artur-tamazian** [comment]
Thank you @mredjem - this is a very clean and well-written change! 
I have one small suggestion after which I can submit and release a new version.

File: src/main/java/com/at/avro/AvroField.java (hunk 1 above)
Comment (mredjem) [reply src/main/java/com/at/avro/AvroField.java:61]: Yes it does return an empty string when comments are missing.

My first thought was to always have the `doc` field, even an empty one, when flag `useSqlCommentsAsDoc` was set to `true`. However as the `doc` field is an optional one in an Avro schema, I could add it only when a comment is present.

Comment (mredjem) [reply src/main/java/com/at/avro/AvroField.java]: Replaced with a call to `Stringutils.isNotBlank`, the `doc` field will be added only when the flag `useSqlCommentAsDoc` is set to `true` and when there exists a non empty comment on the column.

Comment (artur-tamazian) [reply src/main/java/com/at/avro/AvroField.java]: Thanks!

**Pull request merged**